lib/rdf.js

```diff
@@ -222,6 +222,8 @@ function createUpdateRDF(manifest) {
 exports.createUpdateRDF = createUpdateRDF;
 
 function getGUID(type) {
+  if (type.match(/^\{[a-f0-9]{8}-(?:[a-f0-9]{4}-){3}[a-f0-9]{12}\}$/))
```

Comment:
Looking at the results from travis I should add braces to this if statement. Should I squash the fix?